domain/src/main/java/com/bottlerocketstudios/brarchitecture/domain/models/User.kt

```diff
@@ -0,0 +1,12 @@
+package com.bottlerocketstudios.brarchitecture.domain.models
+
+data class User(
+    val username: String?,
+    val nickname: String?,
+    val accountStatus: String?,
+    val displayName: String?,
+    val createdOn: String?,
+    val uuid: String?,
+    val links: Links?,
+    val avatarUrl: String?
```

Comment:
@VanyaHuamanBottleRocket nit: Add trailing comma for less noisy future diffs :)

```suggestion
    val avatarUrl: String?,
```